game-core/src/main/java/games/strategy/triplea/odds/calculator/BattleCalculatorPanel.java

```diff
@@ -1336,7 +1336,8 @@ private void setAttackingUnits(final List<Unit> initialUnits) {
         getAttacker(),
         CollectionUtils.getMatches(
             units,
-            Matches.unitCanBeInBattle(true, isLand(), 1, hasMaxRounds(isLand(), data), false)),
+            Matches.unitCanBeInBattle(
+                true, isLand(), 1, hasMaxRounds(isLand(), data), false, List.of())),
```

Comment:
side-note, not for thsi PR - this API could use work. From the looks of it, most of the parameters are not used. From a caller perspective, it's kind meaningless of an API call, the 'trues', 'falses', redundant call to `isLand()` are certainly minor problems.

I comment on this as we should be careful to avoid making such problems worse when we can, but here I think the horse has already left the barn.